lib/bolt/config.rb

```diff
@@ -155,6 +155,13 @@ def update_from_file(data)
         @puppetfile_config = data['puppetfile'].select { |k, _| PUPPETFILE_OPTIONS.include?(k) }
       end
 
+      # Set global $future if defined
+      if data.key?('future')
```

Comment:
I think after parsing all the config we should set this probably in the CLI class rather than setting it as a side effect of reading the config file.